### DISCUSSION TITLE
怎么联系到作者？

### DISCUSSION THREAD

**Discussion by alany5210** [yume-chan/ya-webadb#439] [Ideas]
怎么联系到作者？

**Comment by yume-chan** [comment]
You can DM me on Twitter. The account is on my profile page.

**Comment by alany5210** [comment]
有没有国内联系方式，咨询一下付费调整



------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Simon ***@***.***&gt;; 
发送时间: 2022年8月23日(星期二) 下午4:07
收件人: ***@***.***&gt;; 
抄送: "Ч仴╃♀ぺ? ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [yume-chan/ya-webadb] 怎么联系到作者？ (Discussion #439)





 
你可以在 Twitter 上给我发 DM。该帐户在我的个人资料页面上。
 
—
直接回复此邮件，在 GitHub 上查看，或取消订阅。
您收到此消息是因为您编写了该主题。消息 ID：<yume-chan/ya-webadb/repo-discussions/439/comments/3454231 @ github 。com&gt;

**Reply by yume-chan** [reply]
I don't have any Chinese IM. Also, I won't have time to work on this project soon, even paid.

**Comment by alany5210** [comment]
好吧，还是非常感谢您的回复！




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "yume-chan/ya-webadb"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年8月24日(星期三) 下午4:24
***@***.***&gt;;
抄送:&nbsp;"Ч仴╃♀ぺ? ***@***.******@***.***&gt;;
主题:&nbsp;Re: [yume-chan/ya-webadb] 怎么联系到作者？ (Discussion #439)





 
I don't have any Chinese IM. Also, I won't have time to work on this project soon, even paid.
